### DISCUSSION TITLE
No Grid Data 2.4.2

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#10511] [2.4]
Security Onion version (2.4.2) in Proxmox.

I'm unable to see the standalone member under Grid View. Everything else seems to be in working order. Any suggestions?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/f5880a58-61c5-48a2-8506-579ac0c8878e)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/96748451-191b-4a18-9215-e482ca4c5b50)

**Comment by dougburks** [comment]
Was this immediately after installation completed?

Did it eventually show up?

**Reply by ColeVan** [reply]
I didn't notice it until a day after installation. Unfortunately it still hasn't populated.

**Reply by dougburks** [reply]
Have you checked the logs at `/opt/so/log/` for any possible clues?

Have you tried logging out of SOC and logging back in?

Have you tried rebooting?

**Comment by ColeVan** [comment]
Have you checked the logs at /opt/so/log/ for any possible clues?

- Yes. I've looked at several different logs for errors but I don't see any that relate to the GRID view.

Have you tried logging out of SOC and logging back in?

- Yes. Several times. Still no changes. I even restarted the soc via sudo so-soc-restart. Still no changes.

Have you tried rebooting?

- Yes. I've rebooted twice but still no changes. I'll keep looking around for more clues.

Here are some screenshots below of what I'm looking at so far.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/e399fd2a-488f-4581-82d1-aa362a8a9965)

**Reply by dougburks** [reply]
Have you looked at `/opt/so/log/sensoroni/sensoroni.log` to see if it's able to check in with the SOC service?

**Reply by ColeVan** [reply]
It would appear that there is no sensoroni.log available to look at. Could this be the issue?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/c105abc6-7410-4552-b399-713c0e6dcc1d)

**Reply by dougburks** [reply]
What is the output of the following?
```
ps aux |grep sensoroni
```

Have you modified anything like hostname, IP address, etc?

**Reply by ColeVan** [reply]
I have SO2 being NAT'd through my pFsense firewall. The internal IP is "10.10.200.65". The soc URL I set it to for NAT is "192.168.1.154"
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/d0b305d0-3327-4871-a494-a5c416ef785a)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/11878d65-35c5-4d07-9d74-3269e30cbc8b)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/a49d8ae7-5adb-48bd-826d-b229102f71df)

**Reply by ColeVan** [reply]
As you can see when I hit 192.168.1.154 it lands me on the SO2 main page. Perhaps the NAT is the issue?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/26f0dca8-2d7e-421f-8f5a-cba99e7bda2a)

**Reply by jertel** [reply]
An alternative configuration option is to choose the hostname option when installing Security Onion, and adjust your DNS as necessary, so web browsers can resolve the hostname to the NAT'd forwarding IP. Meanwhile, the standalone's Sensoroni process will internally resolve the hostname to the manager's actual IP.
